hibernate-core/src/main/java/org/hibernate/bytecode/enhance/spi/interceptor/EnhancementAsProxyLazinessInterceptor.java

```diff
@@ -93,62 +160,94 @@ public Object forceInitialize(Object target, String attributeName) {
 
```

Comment:
can we remove the` BytecodeLogger.LOGGER.tracef`? I think the `forceInitialize` method that is called after does it